Implement entry into GameView from LibraryListItem

arellaTV opened the pull request.

arellaTV committed:
Implement materialize-css

arellaTV committed:
Add GameModel.js

arellaTV committed:
Add id property to LibraryItemView, apply materialize classNames

arellaTV committed:
Incorporate Material Design to LibraryView/LibraryItemView

arellaTV committed:
Bind song id to each libraryItemView

arellaTV committed:
Add container className to LibraryView

arellaTV committed:
Merge branch 'feat/stylize'

arellaTV committed:
Implement Link element in LibraryItemView

arellaTV committed:
Append dummy data

arellaTV committed:
Modify a route to allow the inclusion of param filename

arellaTV committed:
Implement entry into game view from librarylistview

shea-hawkins commented:
Jay, could you please include the minimized and packaged Materialize?

In addition, I see some console logs and debuggers, could you please remove them?

arellaTV committed:
Clean up remaining console logs

arellaTV committed:
Remove unnecessary materialize files

arellaTV commented:
Hi @shea-hawkins! Thanks for the heads up! I made some adjustments that hopefully cleans up a few issues
- removed a lot of unnecessary materialize files in /assets
- cleaned up console.log and debugger (all of them hopefully!)

shea-hawkins commented:
Looks great!  Thanks  Jay
